test/sandbox.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
I'm open to suggestions, but don't want a name specific to sessionStorage issue, this test catches any error when loading QUnit through an iframe.